deepfence_frontend/apps/dashboard/src/features/integrations/components/IntegrationForm.tsx

```diff
@@ -601,10 +74,23 @@ export const IntegrationForm = ({
     return formData?.config?.use_iam_role === 'true';
   });
 
+  const docLink = IntegrationDocsLinkMap[integrationType];
+
   return (
     <>
       {!data?.success ? (
         <fetcher.Form method="post" className="m-4 overflow-y-auto">
+          {docLink ? (
```

Comment:
don't we need to check the length also here, since in original map there are empty strings if there is no url